### PR TITLE
Created config for bug report and GH discussion link

### PR DESCRIPTION
**Pull request by Sophyia7**
@mason-at-pieces I added a bug report configuration in addition to the GH discussion link.

### CODE REVIEW

**Comment by mason-at-pieces** [comment]
So when there is a new release for our obsidian plugin, the files in this repo are updated automatically.

@caleb-at-pieces do you know if a file added, would an update just remove the file or only update the existing files?

**Comment by mason-at-pieces** [comment]
Can we do the same for this as well: https://github.com/pieces-app/obsidian-pieces/pull/20#issuecomment-2004318948

**Comment by Sophyia7** [comment]
@mason-at-pieces resolved! Lmk if i missed anything.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Report Your Bug
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/bug.yml]: 'Report a bug'

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature or Enhancement Request
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/config.yml]: 'Request a Feature'

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    url: https://github.com/pieces-app/support/issues/new?template=bug.yml
+    about: Submit your bug or issues here for the Pieces team to resolve!
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/bug.yml]: 'Report a bug to the pieces team.'

**Review by caleb-at-pieces — CHANGES_REQUESTED** [review]
just some slight copy adjustments and I think we are gtg on this

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+    url: https://github.com/pieces-app/support/discussions
+    about: Post your idea for our Jupyter extension in discussions for the Pieces team and community to discuss!
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/config.yml]: 'Post your idea for the Pieces JupyterLab extension in discussions for the Pieces team and community to discuss!'

Comment (Sophyia7) [reply .github/ISSUE_TEMPLATE/config.yml]: @caleb-at-pieces updated

**Review by caleb-at-pieces — APPROVED** [review]
well done!

**Pull request closed without merging**